docs/blitz/usage/usage.rst

```diff
@@ -0,0 +1,48 @@
+Usage
+=====
+This page contains all the documentation needed to use *Blitz*.
```

Comment:
Now that you have your Trigger Yaml file ready with your blitz testcase(s),  let's get it running. You need the following:

source
tb
trigger yaml

Reply:
add link to your other page which describe how to create the trigger yaml file

Reply:
Done